numpy/add_newdocs.py

```diff
@@ -5919,6 +5919,13 @@ def luf(lamdaexpr, *args, **kwargs):
         First array
     B : array_like
         Second array
+    dtype : data-type code, optional
+        The type used to represent the intermediate results. Defaults
```

Comment:
I don't think this is correct. I believe what happens is that the arrays are converted to the common type, or dtype if given, the operation done, and then converted to the `out` type if the `out` argument is given.